app/src/pages/inside/launchesPage/modals/launchCompareModal/launchCompareModal.jsx

```diff
@@ -32,6 +31,7 @@ import { SpinningPreloader } from 'components/preloaders/spinningPreloader';
 import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
 import { defectColorsSelector } from 'controllers/project';
 import { activeProjectSelector } from 'controllers/user';
+import { C3Chart } from '../../../../../components/widgets/charts/common/c3chart';
```

Comment:
Please use alias for importing components.

Reply:
Fixed.